tests/cnf/core/network/metallb/tests/frrk8-tests.go

```diff
@@ -246,7 +246,7 @@ var _ = Describe("FRR", Ordered, Label(tsparams.LabelFRRTestCases), ContinueOnFa
 				Eventually(func() string {
 					// Get the routes
 					frrNodeState, err := metallb.ListFrrNodeState(APIClient, client.ListOptions{
-						FieldSelector: fields.SelectorFromSet(fields.Set{"metadata.name": "worker-0"})})
+						FieldSelector: fields.SelectorFromSet(fields.Set{"metadata.name": cnfWorkerNodeList[0].Object.Name})})
```

Comment:
Same comment Anvesh left on the 4.18 version
use workerNodeList instead of cnfWorkerNodeList everywhere